setup.cfg

```diff
@@ -114,4 +114,4 @@ format = parsable
+exclusion_regex = ^src/prefect/_vendor/.*$|^src/prefect/utilities/compat\.py$|^src/prefect/engine/flow_runner\.py$|^src/prefect/engine/task_runner\.py$|^src/prefect/engine/executor\.py$|^src/prefect/engine/result\.py$|^src/prefect/engine/runner\.py$|^src/prefect/engine/flow_runner\.py$|^src/prefect/engine/task_runner\.py$|^src/prefect/engine/executor\.py$|^src/prefect/engine/result\.py$|^src/prefect/engine/runner\.py$|^src/prefect/engine/flow_runner\.py$|^src/prefect/engine/task_runner\.py$|^src/prefect/engine/executor\.py$|^src/prefect/engine/result\.py$|^src/prefect/engine/runner\.py$|^src/prefect/engine/flow_runner\.py$|^src/prefect/engine/task_runner\.py$|^src/prefect/engine/executor\.py$|^src/prefect/engine/result\.py$|^src/prefect/engine/runner\.py$|^src/prefect/engine/flow_runner\.py$|^src/prefect/engine/task_runner\.py$|^src/prefect/engine/executor\.py$|^src/prefect/engine/result\.py$|^src/prefect/engine/runner\.py$|^src/prefect/engine/flow_runner\.py$|^src/prefect/engine/task_runner\.py$|^src/prefect/engine/executor\.py$|^src/prefect/engine/result\.py$|^src/prefect/engine/runner\.py$|^src/prefect/engine/flow_runner\.py$|^src/prefect/engine/task_runner\.py$|^src/prefect/engine/executor\.py$|^src/prefect/engine/result\.py$|^src/prefect/engine/runner\.py$|^src/prefect/engine/flow_runner\.py$|^src/prefect/engine/task_runner\.py$|^src/prefect/engine/executor\.py$|^src/prefect/engine/result\.py$|^src/prefect/engine/runner\.py$|^src/prefect/engine/flow_runner\.py$|^src/prefect/engine/task_runner\.py$|^src/prefect/engine/executor\.py$|^src/prefect/engine/result\.py$|^tests/workers/test_process_worker.py$|^tests/test_background_tasks.py|^src/integrations/.*/tests/.*$
```

Comment:
Why wouldn't we need the tests to pass vermin?  They will run on each supported Python version, right?

Reply:
How about this: https://github.com/netromdk/vermin?tab=readme-ov-file#analysis-exclusions

`# novermin`?